### PR TITLE
set num_of_process to a feasible value

### PR DESCRIPTION
**Pull request by PaulNicolasHunter**
w.r.t issue [#311](https://github.com/quark-engine/quark-engine/issues/311), number of process has been set to a value which is less or equal to the cpu_count - 1.

### CODE REVIEW

File: quark/core/parallelquark.py [hunk 1]
```diff
@@ -113,7 +115,8 @@ def _apply_analysis_result(self, rule_obj):
     def __init__(self, apk, core_library, num_of_process=1):
         self._result_map = {}
         self._pool = Pool(
-            num_of_process - 1, self._worker_initializer, (apk, core_library)
+            min(num_of_process, cpu_count - 1), self._worker_initializer,
```
Comment (haeter525) [inline quark/core/parallelquark.py:118]: `cpu_count` is a function. You should call it to get the number of cores.

**Review by haeter525 — CHANGES_REQUESTED** [review]
Hi @PaulNicolasHunter.
I have left a comment on this change. Please test the code first before submitting a PR.

File: quark/core/parallelquark.py (hunk 1 above)
Comment (PaulNicolasHunter) [reply quark/core/parallelquark.py]: thanks, silly mistake.

**Review by haeter525 — APPROVED** [review]

**Comment by haeter525** [comment]
@PaulNicolasHunter, LGTM! Nice work.
Since we mainly develop on branch master, I will change the target branch and solve the conflicts. Thanks!

**Pull request merged**